### PR TITLE
doc: Fix markup in Calculus of Inductive Constructions

### PR DESCRIPTION
**Pull request by bafain**
**Kind:** documentation.

### CODE REVIEW

File: doc/sphinx/language/cic.rst [hunk 1]
```diff
@@ -721,7 +721,8 @@ called the *context of parameters*. Furthermore, we must have that
 each :math:`T` in :math:`(t:T)∈Γ_I` can be written as: :math:`∀Γ_P,∀Γ_{\mathit{Arr}(t)}, S` where
 :math:`Γ_{\mathit{Arr}(t)}` is called the *Arity* of the inductive type t and :math:`S` is called
 the sort of the inductive type t (not to be confused with :math:`\Sort` which is the set of sorts).
-** Examples** The declaration for parameterized lists is:
+
+**Examples** The declaration for parameterized lists is:
```
Comment (Zimmi48) [inline doc/sphinx/language/cic.rst]: It's probably even better to use the `.. example::` environment there. It was sometimes forgotten during the migration.

**Review by Zimmi48 — APPROVED** [review]
Thanks!

File: doc/sphinx/language/cic.rst (hunk 1 above)
Comment (bafain) [reply doc/sphinx/language/cic.rst]: Alright, I added commits for that which should be squashed before merging; or do you want me to create a separate commit that replaces the bold "Example" text with the `example` directive?

**Comment by Zimmi48** [comment]
Rendered here: https://coq.gitlab.io/-/coq/-/jobs/79352918/artifacts/_install_ci/share/doc/coq/sphinx/html/language/cic.html

**Review by Zimmi48 — APPROVED** [review]
Thanks! Can you proceed with auto-squashing / force-pushing? I'll merge this afterwards.

**Pull request merged**